### PR TITLE
Add FSU terms to ModelMetrics

### PR DESCRIPTION
**Pull request by marianoteehan**
### Your checklist for this pull request

Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [X ] Make sure you are requesting to **pull a feature/bugfix branch** (right side). Don't request your master!
- [X ] Make sure tests pass and coverage has not fallen `docker-compose run --rm test`.
- [X] Update the [CHANGELOG.md](../CHANGELOG.md) to describe your changes in a bulleted list under the "Development" section at the top of the changelog. If this section does not exist, create it.
- [x] Make sure code style follows PEP 008 using `docker-compose run --rm blacken`.
- [n/a] Make sure that new functions and classes have inline docstrings and are
  included in [docs/api.rst](../docs/api.rst) for the sphinx build. Please use [numpy-style docstrings](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#google-vs-numpy).
  Sphinx docs can be built with the following command: `docker-compose run --rm --entrypoint="make -C docs html" shell`. Please note and fix any warnings.
- [ ] Make sure that all git commits are have the "Signed-off-by" message for
  the Developer Certificate of Origin. When you're making a commit, just add
  the `-s/--signoff` flag (e.g., `git commit -s`).

### Description

Added additional terms to ModelMetrics() which are used in computing fractional savings uncertainty.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Changelog
 Development
 -----------
 
-* Placeholder
+* Add additional terms into ModelMetricc() class which can be used in fractional savings uncertainy computations.
```
Comment (ssuffian) [inline CHANGELOG.md:7]: I think you've got a small typo there - ModelMetricc() should be ModelMetrics()

**Review by ssuffian — CHANGES_REQUESTED** [review]
Looks great! These metrics that were defined in caltrack have been missing from eemeter until now. 

Before we merge, I think we should try to get the test coverage back up to 100% - I think we just need a test where avg_metrics is not None, and then it looks like there were some missing code coverage lines in io.py from a previous commit that we could fix the tests for as well (a df where tz is None).

Also I can show you how to do the DCO signing - I always forget to do it myself when I'm committing to eemeter - that's why it says 'All checks have failed'.
